### PR TITLE
Update feature eng doc

### PR DESCRIPTION
**Pull request by xuehui1991**

### CODE REVIEW

File: src/nni_manager/training_service/pai/paiK8S/paiK8SData.ts [hunk 1]
```diff
@@ -34,4 +34,4 @@ export const PAI_K8S_TRIAL_COMMAND_FORMAT: string =
 && ls $NNI_SYS_DIR \
 && cd $NNI_SYS_DIR && sh install_nni.sh \
 && python3 -m nni_trial_tool.trial_keeper --trial_command '{6}' --nnimanager_ip '{7}' --nnimanager_port '{8}' \
---nni_manager_version '{9}' --log_collection '{10}'`;
+--nni_manager_version '{9}' --log_collection '{10}'`;
```
Comment (SparkSnail) [inline src/nni_manager/training_service/pai/paiK8S/paiK8SData.ts]: why change this line?

Comment (xuehui1991) [reply src/nni_manager/training_service/pai/paiK8S/paiK8SData.ts]: fixed

File: docs/en_US/FeatureEngineering/Overview.md [hunk 2]
```diff
@@ -6,6 +6,12 @@ For now, we support the following feature selector:
 - [GradientFeatureSelector](./GradientFeatureSelector.md)
 - [GBDTSelector](./GBDTSelector.md)
 
+These selector are suitable for tabular data(which means it doesn't include image, speech and text data).
+
+In addition, those selector only for feature selection. If you want to:
```
Comment (scarlett2018) [inline docs/en_US/FeatureEngineering/Overview.md]: selectors are

File: src/sdk/pynni/nni/nni [hunk 3]
```diff
@@ -0,0 +1 @@
+../src/sdk/pynni/nni
```
Comment (SparkSnail) [inline src/sdk/pynni/nni/nni]: why add this line?

**Pull request merged**